Makefile

```diff
@@ -117,6 +117,13 @@ generate-crds: $(KUSTOMIZE) manifests kustomize ## generate final crds with kust
 	mkdir -p generated-crds
 	$(KUSTOMIZE) build config/crd -o generated-crds # If -o points to a folder, kustomize saves them as several files instead of 1
 
+.PHONY: tag
+tag:
+	@git tag "${TAG}" || (echo "Tag ${TAG} already exists. If you want to retag, delete it manually and re-run this command" && exit 1)
+	@git-chglog --output CHANGELOG.md
+	@git commit -m 'Update CHANGELOG.md' -- CHANGELOG.md
+	@git tag -f "${TAG}"
+
```

Comment:
Thanks for that!

Reply:
Don't we want to sign tags?

Reply:
agree, it should be `git tag -s -a "${TAG}" -f "${TAG}` or similar.

Reply:
added